server/routes/group.js

```diff
@@ -38,6 +38,12 @@ router.route('/:groupName/escalationpolicy')
     groupMiddleware.isGroupAdmin, groupCtrl.updateEscalationPolicy)
   .post(validate(paramValidation.byName), groupCtrl.scheduleEPRotation);
 
+router.route('/:groupName/escalationPolicy/:userId')
```

Comment:
This confuses me a little bit. It seems like this would be better going through one endpoint with a field in the body defining reactivate or activate rather than two separate endpoints. 

DELETE doesn't seem like the correct type for the reactivate either